blog/2023-02-27-yourls-1-9-2.md

```diff
@@ -0,0 +1,44 @@
+---
+title: YOURLS 1.9.2
+authors:
+  - ozh
+tags:
+  - release
+slug: /2023/02/yourls-1-9-2
```

Comment:
```suggestion
```

The previous slug attributes were there for compatibility with the previous blog system.
Unless the more generic `/YYYY/MM/DD/title` is a no-go for you, we can safely drop it 🙂